### ISSUE TITLE
Can `feature/test` branch be deleted?

### ISSUE THREAD

**Issue by antkmsft** [Azure/azure-sdk-for-c#2019]
Updated 6 months ago by Ahson.

https://github.com/Azure/azure-sdk-for-c/branches/all

**Comment by ahsonkhan** [comment]
Yes, it can be deleted. It was only used to test the workflow for manual merge/commit process instead of the only available "squash and merge" when using the GitHub UI.

However, since it is a "feature" branch, it is automatically protected, so will need the help of @danieljurek to delete it :)

https://github.com/Azure/azure-sdk-for-c/tree/feature/test

![image](https://user-images.githubusercontent.com/6527137/145468913-5cb14c8b-554c-41ab-85c8-6c9e5f40ad68.png)

https://github.com/Azure/azure-sdk-for-c/pull/1748

**Comment by danieljurek** [comment]
Deleted.
